css/customLocale.css

```diff
@@ -6,31 +6,33 @@
  *
  */
 
-.highlight {
-	background-color: #A9E2F3; 
+input.valueChanged, textArea.valueChanged {
+	background-color:#FFFF93 !important;
 }
 
-td.input {
-	border-top: 1px solid #999;
-	padding: 15px 10px 5px 10px;
+.customLocale__headerDescription {
+	padding-top: 0.5rem;
+	font-size: 14px;
+	line-height: 1.6em;
 }
 
-input.valueChanged, textArea.valueChanged {
-	background-color:#FFFF93 !important;
+.pkpTable td.customLocale__itemCount {
+	font-size: 12px;
+	padding: 0.5rem 1rem;
 }
 
-input, textArea {
-	background-color:#eee !important;
+.customLocale__cellTable {
+	border: inherit;
+}
+.customLocale__cellTable tr {
+	border-bottom: inherit;
 }
 
-#nextPage {
-	width:150px;
+.customLocale__cellHeader td {
+	font-weight: bold;
 }
 
-#searchString {
-	float:left;
-	width: 520px;
-	margin: 3px 10px 0px 0px;
-	padding: 0;
+textarea.customLocale__fixedSize {
+	resize: none;
```

Comment:
What was the problem this was meant to address? Was a resize just causing a visual problem or was it breaking the UI in some way? Usually it's better to leave the fields resizable so that users can do what they want with it, but we can leave it in if it's causing a serious problem.

Reply:
Being able to resize the textarea was causing some strange formatting (see screenshot). Have you run into something like this before, or should we keep the resize set to none?

<img width="1429" alt="Screen Shot 2020-05-06 at 07 36 18" src="https://user-images.githubusercontent.com/20844845/81190424-ae326680-8f6c-11ea-8319-89195ef08431.png">

Reply:
Oh, does this only apply to the disabled field? In that case I think you can leave it in...

Reply:
No, it can happen with the input textarea as well.

![Screen Shot 2020-05-06 at 09 33 09](https://user-images.githubusercontent.com/20844845/81203702-18530780-8f7d-11ea-83ee-60ec39d63ae7.png)

Reply:
Hmm, ok leave it as-is for now.

Reply:
Okay, will do. Thanks.